cast-dssspell.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+set -e
+
+[[ "$(seth chain --rpc-url="$ETH_RPC_URL")" == "goerli" ]] || { echo "Please set a Goerli ETH_RPC_URL"; exit 1; }
+
+CHANGELOG=0xdA0Ab1e0017DEbCd72Be8599041a2aa3bA7e740F
+MCD_ADM=$(seth call "$CHANGELOG" 'getAddress(bytes32)(address)' "$(seth --to-bytes32 "$(seth --from-ascii "MCD_ADM")")")
+MCD_GOV=$(seth call "$CHANGELOG" 'getAddress(bytes32)(address)' "$(seth --to-bytes32 "$(seth --from-ascii "MCD_GOV")")")
+MCD_IOU=$(seth call "$CHANGELOG" 'getAddress(bytes32)(address)' "$(seth --to-bytes32 "$(seth --from-ascii "MCD_IOU")")")
+TEN_THOUSAND=$(seth --to-wei 10000 ether)
```

Comment:
This is already 100,001 MKR from our deployer, we should increase this.

Reply:
perhaps name the variable generically too.

Reply:
done in 33cf988

Reply:
i think you missed a `0` based on @godsflaw 's comment.  However, I thought the flow we wanted here was that the deployer holds the MKR, votes, schedules, casts the spell then pulls the MKR back so we don't have to worry about overpowering some lingering MKR from a different address

Reply:
@gbalabasquer?